bash/functions

```diff
@@ -0,0 +1,10 @@
+delete_branches_except() {
```

Comment:
@calina-c: This function looks very helpful. However, I prefer it moved under the `git aliases` file in order to keep all aliases/functions related to a technology in the same place.